code/ui/blocks/src/blocks/Canvas.stories.tsx

```diff
@@ -81,3 +83,41 @@ export const WithAdditionalActions: Story = {
     ],
   },
 };
+
+export const ClassName: Story = {
+  name: 'ClassName',
+  args: {
+    className: 'my-custom-classname',
+  },
+  render: (args) => (
+    <>
+      <p>
+        This story sets the <code>className</code> prop on the <code>Canvas</code> to{' '}
+        <code>my-custom-classname</code>, which will propagate to the preview element. To
+        demonstrate this, it also adds a <code>style</code> tag that sets another background color
+        for that class:
+      </p>
+      <Source
+        language="jsx"
+        code={dedent`<style>
+        .my-custom-classname {
+          background-color: #fd5c9355;
+        }
+      </style>
+      <Canvas className="my-custom-classname">
+        <StoryComponent of={BooleanStories.Undefined} />
+      </Canvas>`}
+      />
+      <style>
```

Comment:
This is an interesting thing you are doing here.

It seems like two things:

1. Documenting what the story is intended for as it's not self explanatory (fair enough). Would a code comment be enough here? Perhaps there should be a SB addon/feature that surfaces such story-level comments in the addons panel somehow.

2. Showing the actual source code of the story. Surely the source addon should be doing this for us? Or is there some reason that doesn't work in this case?

I also wonder if you'd prefer a MDX file for this documentation. It's all a bit mind boggling as we work on docs for docs in docs etc etc.

Reply:
1. You're describing my and Michael's vision for the `Description` block here, being able to describe any comments on stories, meta or components. Doesn't work here though, I think it only works in Docs.
2. I hadn't tried the `storysource` addon, but it worked. I've done that instead:
![image](https://user-images.githubusercontent.com/5678122/203558688-56aa5cb6-0ba1-4c3c-a36c-dbe45d442947.png)


I'm not sure about the MDX file, I'd like there to be some consistency, but that might be unfeasable.

Reply:
> You're describing my and Michael's vision for the Description block here, being able to describe any comments on stories, meta or components. Doesn't work here though, I think it only works in Docs.

Ahh, so we need the description block to also function as an addon (similar to the args table and the source). This is something that we want to do, I think @shilman has a name he uses for that, something like "standalone blocks". 👍

Reply:
yeah he told me about a pretty great vision for it, where any addon could be a block, or vice versa.
Someday. To the moon. 🚀

Reply:
"Docs Unbundling" -- that's it.